### ISSUE TITLE
scoreboard: surface treatment: LED recess

### ISSUE THREAD

**Issue by pluschris** [bigdisp/scoreboard-hardware#17]
@gn36: Is it still up to date that we'll use a router for the LED recesses? Could you clarify if you'll get one? Otherwise I could ask a friend of mine...

**Comment by gn36** [comment]
Yes, I can get one. We have to make sure that the recesses are not too deep, otherwise we might limit the viewing angle of the board. Also we have to make sure that we angle the bottom of the recesses so excess water can leave the recess.
